Build/Scripts/seajax_v2.py

```diff
@@ -60,9 +60,10 @@
     "image": ["standalone"],
     "zoom": ["standalone"],
     "zoomimage": ["standalone"],
-    "pivot": ["standalone"],
     "ajax": ["standalone"],
     "utils": ["standalone"],
+    "pivot": ["standalone"],
+    "collegevine": ["standalone"],
```

Comment:
Create new `collegevine` build target that compiles our forked `App.jsx` (from `PivotView.js`).